### PR TITLE
Stories for layers

### PR DESCRIPTION
**Pull request by alebinson**
Related Issues:
#47 

Closes issues: 
#47 

## Checklist
- [ ] Tests
- [X] Stories
- [ ] Lint
- [ ] Prettier

## What I did

- [ ] BREAKING CHANGE
- [ ] feature request
- [ ] bug fix
- [X] documentation

More information:

### CODE REVIEW

**Comment by alebinson** [comment]
Find a way to fix Linting warnings about return type (making another PR)
export const XYZ **: CSFStory<JSX.Element>** = () => ();

File: packages/react-components/src/utils/story.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+export interface CSFStory<StoryFnReturnType = unknown> {
+  story?: {
+    name?: string;
+    decorators?: DecoratorFunction<StoryFnReturnType>[];
+    parameters?: { [name: string]: unknown };
+  };
+  argTypes?: ArgTypes;
+  (): StoryFnReturnType;
+};
```
Comment (syncush) [inline packages/react-components/src/utils/story.ts]: Currently `tsbuildconfig.json` `rootDir` is `src/lib`, utils dir is a sibling of the `src\lib` folder, `lerna bootstrap` fails

**Review by syncush — CHANGES_REQUESTED** [review]

**Review by syncush — CHANGES_REQUESTED** [review]
build and test fails on tsbuildconfig.json

File: packages/react-components/src/utils/story.ts (hunk 1 above)
Comment (alebinson) [reply packages/react-components/src/utils/story.ts]: @syncush utlils moved under lib

**Review by syncush — APPROVED** [review]

**Pull request merged**